src/gmail-ai-classifier/tests/code.test.js

```
@@ -0,0 +1,17 @@
+/**
+ * Unit tests for Gmail AI Classifier
+ */
+
+describe('Gmail AI Classifier Config', () => {
+  test('returns standard config schema', () => {
+    const config = {
+      geminiApiKey: 'TEST_KEY',
+      processedLabel: 'Processed',
+      autoFilterConfidenceThreshold: 0.95,
+      canonicalDomains: ['01_Household/Primary_House', '04_Family_Health/Bowens']
```

ESLint

[error] Replace `'01_Household/Primary_House',·'04_Family_Health/Bowens']` with `⏎········'01_Household/Primary_House',⏎········'04_Family_Health/Bowens',⏎······],`

(prettier/prettier)